src/app/launcher/service/mission-runtime.service.ts

```diff
@@ -30,7 +30,7 @@ export abstract class MissionRuntimeService {
     if (!cluster) {
       return { empty: false, boosters: availableBoosters };
     }
-    const boostersRunningOnCluster = boosters.filter(b => MissionRuntimeService.checkRunsOnCluster(b, cluster));
+    const boostersRunningOnCluster = availableBoosters.filter(b => MissionRuntimeService.checkRunsOnCluster(b, cluster));
```

Comment:
@edewit this was the bug, and it was a tricky one :)

Reply:
I see, you wouldn't see this if non available booster wouldn't run on your cluster